vpr/src/draw/search_bar.cpp

```diff
@@ -96,14 +98,33 @@ void search_and_highlight(GtkWidget* /*widget*/, ezgl::application* app) {
             return;
         }
 
-        highlight_blocks((ClusterBlockId)block_id);
+        highlight_cluster_block((ClusterBlockId)block_id);
     }
 
     else if (search_type == "Block Name") {
         std::string block_name = "";
         ss >> block_name;
 
-        highlight_blocks((std::string)block_name);
+        AtomBlockId atom_blk_id = AtomBlockId::INVALID();
+        atom_blk_id = atom_ctx.nlist.find_block(block_name);
```

Comment:
Do we need to set this to INVALID? Couldn't we just collapse these two lines into one, as the block_id is updated no matter what?